recipes/metpy/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: metpy
+  version: !!str 0.3.0
+
+source:
+  fn: metpy-0.3.0.tar.gz
+  url: https://pypi.python.org/packages/source/M/MetPy/MetPy-0.3.0.tar.gz
+  md5: efa24aea506b2d0a829035aea94e6d89
+
```

Comment:
Add a build field like:

```
build:
    script: python setup.py install --single-version-externally-managed --record record.txt
    number: 0
```

So we can track the build number. (The build script will soon replace `bld.bat` and `build.sh`.)